torch/nn/modules/batchnorm.py

```diff
@@ -23,8 +23,8 @@ def __init__(self, num_features, eps=1e-5, momentum=0.1, affine=True,
         self.num_features = num_features
         self.eps = eps
         self.momentum = momentum
-        self.affine = affine
-        self.track_running_stats = track_running_stats
+        self._affine = affine
```

Comment:
i am wondering if this renaming will have implications for loading old state_dicts, didn't look into the code deeply yet

Reply:
Oh, I didn't think about the state_dict migration issue. Thanks for letting me know.